target/linux/ramips/dts/mt7628an_tplink_re200-v2.dts

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an_tplink_re200.dtsi"
+
+/ {
+	compatible = "tplink,re200-v2", "mediatek,mt7628an-soc";
+	model = "TP-Link RE200 v2";
+};
+
+&leds {
+	wifi2g_red {
+		label = "re200-v2:red:wifi2g";
+		gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+	};
+
+	wifi5g_red {
+		label = "re200-v2:red:wifi5g";
+		gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&led_wps {
+	label = "re200-v2:green:wps";
+};
+
+&led_wifi {
+	label = "re200-v2:green:wifi";
+};
+
+&led_lan {
+	label = "re200-v2:green:lan";
+};
+
+&led_power {
+	label = "re200-v2:green:power";
+};
+
+&led_wifi2g_green {
+	label = "re200-v2:green:wifi2g";
+};
+
+&led_wifi5g_green {
+	label = "re200-v2:green:wifi5g";
+};
```

Comment:
I'm not a fan of this approach. I'd just move the whole LED block to the DTS files, it's much easier to read, and the few duplicate lines won't kill us.

Reply:
And if you have certain LEDs only present at one variant, you will have to move state_default, too.